### ISSUE TITLE
Installation in Python 3.7.1 leads to modules not found errors

### ISSUE THREAD

**Issue by joseortiz3** [wgurecky/StarVine#16]
Hello, installed your package on windows using the minGW fortran compiler, Python 3.7.1.

Importing results in:
```python
>>> import starvine

Traceback (most recent call last):
  File "C:\Users\Joey\AppData\Local\Programs\Python\Python37\lib\site-packages\starvine-0.0.1-py3.7-win-amd64.egg\starvine\bvcopula\copula_factory.py", line 4, in <module>
    from copula import *
ModuleNotFoundError: No module named 'copula'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Joey\AppData\Local\Programs\Python\Python37\lib\site-packages\starvine-0.0.1-py3.7-win-amd64.egg\starvine\__init__.py", line 3, in <module>
    import starvine.bvcopula
  File "C:\Users\Joey\AppData\Local\Programs\Python\Python37\lib\site-packages\starvine-0.0.1-py3.7-win-amd64.egg\starvine\bvcopula\__init__.py", line 4, in <module>
    from .copula_factory import Copula
  File "C:\Users\Joey\AppData\Local\Programs\Python\Python37\lib\site-packages\starvine-0.0.1-py3.7-win-amd64.egg\starvine\bvcopula\copula_factory.py", line 6, in <module>
    from starvine.bvcopula.copula import *
  File "C:\Users\Joey\AppData\Local\Programs\Python\Python37\lib\site-packages\starvine-0.0.1-py3.7-win-amd64.egg\starvine\bvcopula\copula\t_copula.py", line 7, in <module>
    from copula_base import CopulaBase
ModuleNotFoundError: No module named 'copula_base'
```

The installation appeared to work fine, so I'm not sure why this is happening:
```javascript
Z:\Software\StarVine>python setup.py install
C:\Users\Joey\AppData\Local\Programs\Python\Python37\lib\distutils\dist.py:274: UserWarning: Unknown distribution option: 'build_requires'
  warnings.warn(msg)
Appending starvine.bvcopula.copula.mvtdstpack configuration to starvine.bvcopula.copula
Ignoring attempt to set 'name' (from 'starvine.bvcopula.copula' to 'starvine.bvcopula.copula.mvtdstpack')
Appending starvine.bvcopula.copula configuration to starvine.bvcopula
Ignoring attempt to set 'name' (from 'starvine.bvcopula' to 'starvine.bvcopula.copula')
Appending starvine.bvcopula configuration to starvine
Ignoring attempt to set 'name' (from 'starvine' to 'starvine.bvcopula')
Appending starvine.mvar configuration to starvine
Ignoring attempt to set 'name' (from 'starvine' to 'starvine.mvar')
Appending starvine.uvar configuration to starvine
Ignoring attempt to set 'name' (from 'starvine' to 'starvine.uvar')
Appending starvine.vine configuration to starvine
Ignoring attempt to set 'name' (from 'starvine' to 'starvine.vine')
running install
running bdist_egg
running egg_info
running build_src
build_src
building py_modules sources
building extension "starvine.bvcopula.copula.mvtdstpack.mvtdstpack" sources
f2py options: []
  adding 'build\src.win-amd64-3.7\build\src.win-amd64-3.7\starvine\bvcopula\copula\mvtdstpack\fortranobject.c' to sources.
  adding 'build\src.win-amd64-3.7\build\src.win-amd64-3.7\starvine\bvcopula\copula\mvtdstpack' to include_dirs.
  adding 'build\src.win-amd64-3.7\starvine\bvcopula\copula\mvtdstpack\mvtdstpack-f2pywrappers.f' to sources.
building data_files sources
build_src: building npy-pkg config files
writing StarVine.egg-info\PKG-INFO
writing dependency_links to StarVine.egg-info\dependency_links.txt
writing requirements to StarVine.egg-info\requires.txt
writing top-level names to StarVine.egg-info\top_level.txt
reading manifest file 'StarVine.egg-info\SOURCES.txt'
writing manifest file 'StarVine.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
copying build\src.win-amd64-3.7\starvine\__config__.py -> build\lib.win-amd64-3.7\starvine
running build_ext
No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
customize MSVCCompiler
customize MSVCCompiler using build_ext
get_default_fcompiler: matching types: '['gnu', 'intelv', 'absoft', 'compaqv', 'intelev', 'gnu95', 'g95', 'intelvem', 'intelem', 'flang']'
customize GnuFCompiler
Could not locate executable g77
Could not locate executable f77
customize IntelVisualFCompiler
Could not locate executable ifort
Could not locate executable ifl
customize AbsoftFCompiler
Could not locate executable f90
customize CompaqVisualFCompiler
Could not locate executable DF
customize IntelItaniumVisualFCompiler
Could not locate executable efl
customize Gnu95FCompiler
Found executable C:\MinGW\bin\gfortran.exe
customize Gnu95FCompiler
customize Gnu95FCompiler using build_ext
copying build\temp.win-amd64-3.7\Release\.libs\libmvtdstpa.3PEPBPPM6X2YPXRN6K2LIK3BGKI5UAE4.gfortran-win_amd64.dll -> build\lib.win-amd64-3.7\starvine\.libs
creating build\bdist.win-amd64\egg
creating build\bdist.win-amd64\egg\starvine
creating build\bdist.win-amd64\egg\starvine\.libs
copying build\lib.win-amd64-3.7\starvine\.libs\libmvtdstpa.3PEPBPPM6X2YPXRN6K2LIK3BGKI5UAE4.gfortran-win_amd64.dll -> build\bdist.win-amd64\egg\starvine\.libs
creating build\bdist.win-amd64\egg\starvine\bvcopula
copying build\lib.win-amd64-3.7\starvine\bvcopula\bv_plot.py -> build\bdist.win-amd64\egg\starvine\bvcopula
creating build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\clayton_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\copula_base.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\copula_rotation.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\frank_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\gauss_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\gumbel_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\indep_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\marshall_olkin_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
creating build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\mvtdstpack\mvtdstpack.cp37-win_amd64.pyd -> build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\mvtdstpack\mvtdstpack_license.txt -> build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\mvtdstpack\setup.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\mvtdstpack\__init__.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\setup.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\t_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula\__init__.py -> build\bdist.win-amd64\egg\starvine\bvcopula\copula
copying build\lib.win-amd64-3.7\starvine\bvcopula\copula_factory.py -> build\bdist.win-amd64\egg\starvine\bvcopula
copying build\lib.win-amd64-3.7\starvine\bvcopula\pc_base.py -> build\bdist.win-amd64\egg\starvine\bvcopula
copying build\lib.win-amd64-3.7\starvine\bvcopula\setup.py -> build\bdist.win-amd64\egg\starvine\bvcopula
copying build\lib.win-amd64-3.7\starvine\bvcopula\stat_tests.py -> build\bdist.win-amd64\egg\starvine\bvcopula
copying build\lib.win-amd64-3.7\starvine\bvcopula\__init__.py -> build\bdist.win-amd64\egg\starvine\bvcopula
creating build\bdist.win-amd64\egg\starvine\mvar
copying build\lib.win-amd64-3.7\starvine\mvar\mvd.py -> build\bdist.win-amd64\egg\starvine\mvar
copying build\lib.win-amd64-3.7\starvine\mvar\mv_plot.py -> build\bdist.win-amd64\egg\starvine\mvar
copying build\lib.win-amd64-3.7\starvine\mvar\setup.py -> build\bdist.win-amd64\egg\starvine\mvar
copying build\lib.win-amd64-3.7\starvine\mvar\__init__.py -> build\bdist.win-amd64\egg\starvine\mvar
copying build\lib.win-amd64-3.7\starvine\setup.py -> build\bdist.win-amd64\egg\starvine
creating build\bdist.win-amd64\egg\starvine\uvar
copying build\lib.win-amd64-3.7\starvine\uvar\setup.py -> build\bdist.win-amd64\egg\starvine\uvar
copying build\lib.win-amd64-3.7\starvine\uvar\uvd.py -> build\bdist.win-amd64\egg\starvine\uvar
copying build\lib.win-amd64-3.7\starvine\uvar\uvmodel_factory.py -> build\bdist.win-amd64\egg\starvine\uvar
copying build\lib.win-amd64-3.7\starvine\uvar\__init__.py -> build\bdist.win-amd64\egg\starvine\uvar
creating build\bdist.win-amd64\egg\starvine\vine
copying build\lib.win-amd64-3.7\starvine\vine\base_vine.py -> build\bdist.win-amd64\egg\starvine\vine
copying build\lib.win-amd64-3.7\starvine\vine\C_vine.py -> build\bdist.win-amd64\egg\starvine\vine
copying build\lib.win-amd64-3.7\starvine\vine\D_vine.py -> build\bdist.win-amd64\egg\starvine\vine
copying build\lib.win-amd64-3.7\starvine\vine\setup.py -> build\bdist.win-amd64\egg\starvine\vine
copying build\lib.win-amd64-3.7\starvine\vine\tree.py -> build\bdist.win-amd64\egg\starvine\vine
copying build\lib.win-amd64-3.7\starvine\vine\__init__.py -> build\bdist.win-amd64\egg\starvine\vine
copying build\lib.win-amd64-3.7\starvine\__config__.py -> build\bdist.win-amd64\egg\starvine
copying build\lib.win-amd64-3.7\starvine\__init__.py -> build\bdist.win-amd64\egg\starvine
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\bv_plot.py to bv_plot.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\clayton_copula.py to clayton_copula.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\copula_base.py to copula_base.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\copula_rotation.py to copula_rotation.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\frank_copula.py to frank_copula.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\gauss_copula.py to gauss_copula.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\gumbel_copula.py to gumbel_copula.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\indep_copula.py to indep_copula.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\marshall_olkin_copula.py to marshall_olkin_copula.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack\setup.py to setup.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack\__init__.py to __init__.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\setup.py to setup.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\t_copula.py to t_copula.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\__init__.py to __init__.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula_factory.py to copula_factory.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\pc_base.py to pc_base.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\setup.py to setup.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\stat_tests.py to stat_tests.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\__init__.py to __init__.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\mvar\mvd.py to mvd.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\mvar\mv_plot.py to mv_plot.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\mvar\setup.py to setup.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\mvar\__init__.py to __init__.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\setup.py to setup.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\uvar\setup.py to setup.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\uvar\uvd.py to uvd.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\uvar\uvmodel_factory.py to uvmodel_factory.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\uvar\__init__.py to __init__.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\vine\base_vine.py to base_vine.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\vine\C_vine.py to C_vine.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\vine\D_vine.py to D_vine.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\vine\setup.py to setup.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\vine\tree.py to tree.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\vine\__init__.py to __init__.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\__config__.py to __config__.cpython-37.pyc
byte-compiling build\bdist.win-amd64\egg\starvine\__init__.py to __init__.cpython-37.pyc
creating stub loader for starvine\bvcopula\copula\mvtdstpack\mvtdstpack.cp37-win_amd64.pyd
byte-compiling build\bdist.win-amd64\egg\starvine\bvcopula\copula\mvtdstpack\mvtdstpack.py to mvtdstpack.cpython-37.pyc
installing package data to build\bdist.win-amd64\egg
running install_data
creating build\bdist.win-amd64\egg\starvine\bvcopula\tests
creating build\bdist.win-amd64\egg\starvine\bvcopula\tests\data
copying starvine\bvcopula\tests\data\stocks.csv -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\data
copying starvine\bvcopula\tests\test_ktau_func.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_stats.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_t_copula_fit.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_multifit.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_sample_copula.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_gauss_rotate.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_ktau_fit.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_copula_cdf.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_freeze_params.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\test_rotation.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
copying starvine\bvcopula\tests\__init__.py -> build\bdist.win-amd64\egg\starvine\bvcopula\tests\
creating build\bdist.win-amd64\egg\starvine\mvar\tests
copying starvine\mvar\tests\test_pca.py -> build\bdist.win-amd64\egg\starvine\mvar\tests\
copying starvine\mvar\tests\__init__.py -> build\bdist.win-amd64\egg\starvine\mvar\tests\
creating build\bdist.win-amd64\egg\starvine\uvar\tests
copying starvine\uvar\tests\test_gaussian_fit.py -> build\bdist.win-amd64\egg\starvine\uvar\tests\
copying starvine\uvar\tests\test_beta_fit.py -> build\bdist.win-amd64\egg\starvine\uvar\tests\
copying starvine\uvar\tests\test_gamma_fit.py -> build\bdist.win-amd64\egg\starvine\uvar\tests\
copying starvine\uvar\tests\__init__.py -> build\bdist.win-amd64\egg\starvine\uvar\tests\
creating build\bdist.win-amd64\egg\starvine\vine\tests
copying starvine\vine\tests\__init__.py -> build\bdist.win-amd64\egg\starvine\vine\tests\
copying license.txt -> build\bdist.win-amd64\egg\starvine
copying requirements.txt -> build\bdist.win-amd64\egg\starvine
running install_clib
No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
customize MSVCCompiler
creating build\bdist.win-amd64\egg\EGG-INFO
copying StarVine.egg-info\PKG-INFO -> build\bdist.win-amd64\egg\EGG-INFO
copying StarVine.egg-info\SOURCES.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying StarVine.egg-info\dependency_links.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying StarVine.egg-info\requires.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying StarVine.egg-info\top_level.txt -> build\bdist.win-amd64\egg\EGG-INFO
writing build\bdist.win-amd64\egg\EGG-INFO\native_libs.txt
zip_safe flag not set; analyzing archive contents...
starvine.__pycache__.__config__.cpython-37: module references __file__
starvine.bvcopula.copula.mvtdstpack.__pycache__.mvtdstpack.cpython-37: module references __file__
creating 'dist\StarVine-0.0.1-py3.7-win-amd64.egg' and adding 'build\bdist.win-amd64\egg' to it
removing 'build\bdist.win-amd64\egg' (and everything under it)
Processing StarVine-0.0.1-py3.7-win-amd64.egg
removing 'c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\StarVine-0.0.1-py3.7-win-amd64.egg' (and everything under it)
error removing c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\StarVine-0.0.1-py3.7-win-amd64.egg: [WinError 145] The directory is not empty: 'c:\\users\\joey\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\StarVine-0.0.1-py3.7-win-amd64.egg'
Extracting StarVine-0.0.1-py3.7-win-amd64.egg to c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
StarVine 0.0.1 is already the active version in easy-install.pth

Installed c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\starvine-0.0.1-py3.7-win-amd64.egg
Processing dependencies for StarVine==0.0.1
Searching for numba==0.41.0
Best match: numba 0.41.0
Processing numba-0.41.0-py3.7-win-amd64.egg
numba 0.41.0 is already the active version in easy-install.pth
Installing numba script to C:\Users\Joey\AppData\Local\Programs\Python\Python37\Scripts
Installing pycc script to C:\Users\Joey\AppData\Local\Programs\Python\Python37\Scripts

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\numba-0.41.0-py3.7-win-amd64.egg
Searching for statsmodels==0.9.0
Best match: statsmodels 0.9.0
Processing statsmodels-0.9.0-py3.7-win-amd64.egg
statsmodels 0.9.0 is already the active version in easy-install.pth

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\statsmodels-0.9.0-py3.7-win-amd64.egg
Searching for emcee==3.0rc2
Best match: emcee 3.0rc2
Processing emcee-3.0rc2-py3.7.egg
emcee 3.0rc2 is already the active version in easy-install.pth

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\emcee-3.0rc2-py3.7.egg
Searching for networkx==2.2
Best match: networkx 2.2
Processing networkx-2.2-py3.7.egg
networkx 2.2 is already the active version in easy-install.pth

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\networkx-2.2-py3.7.egg
Searching for seaborn==0.9.0
Best match: seaborn 0.9.0
Adding seaborn 0.9.0 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for h5py==2.9.0rc1
Best match: h5py 2.9.0rc1
Processing h5py-2.9.0rc1-py3.7-win-amd64.egg
h5py 2.9.0rc1 is already the active version in easy-install.pth

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\h5py-2.9.0rc1-py3.7-win-amd64.egg
Searching for pandas==0.23.4
Best match: pandas 0.23.4
Adding pandas 0.23.4 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for scipy==1.1.0
Best match: scipy 1.1.0
Adding scipy 1.1.0 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for numpy==1.15.3
Best match: numpy 1.15.3
Adding numpy 1.15.3 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for llvmlite==0.26.0
Best match: llvmlite 0.26.0
Processing llvmlite-0.26.0-py3.7-win-amd64.egg
llvmlite 0.26.0 is already the active version in easy-install.pth

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\llvmlite-0.26.0-py3.7-win-amd64.egg
Searching for patsy==0.5.1
Best match: patsy 0.5.1
Processing patsy-0.5.1-py3.7.egg
patsy 0.5.1 is already the active version in easy-install.pth

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages\patsy-0.5.1-py3.7.egg
Searching for decorator==4.3.0
Best match: decorator 4.3.0
Adding decorator 4.3.0 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for matplotlib==3.0.1
Best match: matplotlib 3.0.1
Adding matplotlib 3.0.1 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for six==1.11.0
Best match: six 1.11.0
Adding six 1.11.0 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for python-dateutil==2.7.5
Best match: python-dateutil 2.7.5
Adding python-dateutil 2.7.5 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for pytz==2018.6
Best match: pytz 2018.6
Adding pytz 2018.6 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for pyparsing==2.2.2
Best match: pyparsing 2.2.2
Adding pyparsing 2.2.2 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for cycler==0.10.0
Best match: cycler 0.10.0
Adding cycler 0.10.0 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for kiwisolver==1.0.1
Best match: kiwisolver 1.0.1
Adding kiwisolver 1.0.1 to easy-install.pth file

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Searching for setuptools==39.0.1
Best match: setuptools 39.0.1
Adding setuptools 39.0.1 to easy-install.pth file
Installing easy_install-script.py script to C:\Users\Joey\AppData\Local\Programs\Python\Python37\Scripts
Installing easy_install.exe script to C:\Users\Joey\AppData\Local\Programs\Python\Python37\Scripts
Installing easy_install-3.6-script.py script to C:\Users\Joey\AppData\Local\Programs\Python\Python37\Scripts
Installing easy_install-3.6.exe script to C:\Users\Joey\AppData\Local\Programs\Python\Python37\Scripts

Using c:\users\joey\appdata\local\programs\python\python37\lib\site-packages
Finished processing dependencies for StarVine==0.0.1
```

**Comment by wgurecky** [comment]
Thanks for checking this out.  StarVine is currently only tested with python2.7 (CI here https://travis-ci.org/wgurecky/StarVine) .  I will have some time in the next few week to significantly modernize the code base and make sure this works with python3.x!

**Comment by joseortiz3** [comment]
Thanks for the reply @wgurecky. Can't wait to try it out. Amazing how few packages for anything related to copulas there are for Python. Still looking for vine functionality compatible with py37, but looks like I'm going to have to switch to R for this task for now (finance). Thanks again.

**Comment by wgurecky** [comment]
@joseortiz3 @NineIK  Thank you for your work on this.  I used a few of your fixes in PR #19 in addition to a few other small adjustments.
